conans/client/conan_api.py

```diff
@@ -443,13 +445,14 @@ def install_reference(self, reference, settings=None, options=None, env=None,
             generators = False
 
         mkdir(install_folder)
-        self._manager.install(reference=reference, install_folder=install_folder, remote_name=remote,
-                              profile=profile, build_modes=build, update=update,
-                              manifest_folder=manifest_folder,
-                              manifest_verify=manifest_verify,
-                              manifest_interactive=manifest_interactive,
-                              generators=generators,
-                              install_reference=True)
+        ret = self._manager.install(reference=reference, install_folder=install_folder,
+                                    remote_name=remote, profile=profile, build_modes=build,
+                                    update=update, manifest_folder=manifest_folder,
+                                    manifest_verify=manifest_verify,
+                                    manifest_interactive=manifest_interactive,
+                                    generators=generators, install_reference=True)
+        if ret is None:
+            return self._recorder.get_info()
```

Comment:
The export_pkg doesn't have return defined yet. Remove the all the ret stuff and do not return anything.